docs/site-generator/hugo-template/Makefile

```diff
@@ -0,0 +1,78 @@
+# Default shell
+SHELL := /bin/bash
+
+# Config
+DESTINATION ?= public
+LANDING_PATH ?=
+
+# System
+url_os = 64bit
```

Comment:
https://github.com/src-d/ci/blob/master/docs/init.d/init.sh#L22-L32

The makes should **not** install anything. All installations should be done via the init script. It's all in the same repo, so if something changes in `site-generator` one knows that installation scripts need to be updated.